doc/classes/StandardMaterial3D.xml

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="StandardMaterial3D" inherits="BaseMaterial3D" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		PBR Material that can be applied to 3D objects.
```

Comment:
May wish to say: "Physically based rendering (PBR) Material that can be applied to 3D objects."